views/js/controller/runtime/testRunner.js

```diff
@@ -942,6 +969,9 @@ define([
                 TestRunner.beforeTransition();
                 TestRunner.testContext = testContext;
 
+                TestRunner.keepItemTimed(TestRunner.getSessionStateService().getDuration());
+                TestRunner.getSessionStateService().restart();
```

Comment:
The getSessionStateService() return an object with getDuration and reset method but no restart one